Gemfile

```diff
@@ -5,34 +5,34 @@ ruby '2.5.5'
 
 gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
 
-gem 'pg', '~> 1.1'
+gem 'pg',    '~> 1.1'
 
-gem 'puma', '~> 5.0'
-
-group :development, :test do
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
-end
+gem 'puma',  '~> 5.0'
 
 group :development do
   gem 'listen', '~> 3.3'
 end
 
 group :test do
-  # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '>= 3.26'
-  gem 'selenium-webdriver'
-  # Easy installation and use of web drivers to run system tests with browsers
-  gem 'webdrivers'
+  gem 'json_matchers',    '~> 0.11.1'
+  gem 'shoulda-matchers', '~> 4.0'
 end
 
 group :development, :test do
-  gem 'brakeman'
-  gem 'bundler-audit', '~> 0.8.0'
-  gem 'fasterer'
-  gem 'overcommit'
-  gem 'rails_best_practices'
-  gem 'reek'
-  gem 'rubocop', require: false
+  gem 'brakeman',             '~> 5.0'
+  gem 'bundler-audit',        '~> 0.8.0'
+  gem 'faker', git: 'https://github.com/faker-ruby/faker.git'
+  gem 'fasterer',             '~> 0.9.0'
+  gem 'overcommit',           '~> 0.57.0'
+  gem 'pry-byebug',           '~> 3.9.0'
+  gem 'pry-rails',            '~> 0.3.9'
+  gem 'rails_best_practices', '~> 1.20.1'
+  gem 'reek',                 '~> 6.0.4'
+  gem 'rspec-rails',          '~> 5.0.0'
+  gem 'rubocop',              '~> 1.15.0', require: false
```

Comment:
Seems like not full rubocop staff required. Please also add `rubocop-performance`, `rubocop-rails` and `rubocop-rspec`.